Syntax.in.txt

```diff
@@ -0,0 +1,39 @@
+﻿<relation>  :: = < | <= | == | != | >= | > | && |  | |  
+<operation> :: = + | - | * | / | % | ^
+<primary_type> :: = number | string | bool
+<array_type> :: = <primary_type> [ ]
+<type> :: = <primary_type> | <array_type>
+<declaration> :: = let <id> : <type> ;
+<declaration_list> :: = <declaration> | <declaration><declaration_list>
+
+
+<constant> :: = <number> | <string> | <bool>
+<term> :: = <id> | <constant>
+<term_list> :: = <term> | <term><term_list>;

```

Comment:
Add a space between <term> and <term_list> (so that you have "4 5 ab;" instead of "45ab;")